src/components/playback/playbackmanager.js

```diff
@@ -1463,6 +1487,24 @@ class PlaybackManager {
             return getPlayerData(player).subtitleStreamIndex;
         };
 
+        self.getSecondarySubtitleStreamIndex = function (player) {
+            player = player || self._currentPlayer;
+
+            try {
+                if (player && !enableLocalPlaylistManagement(player)) {
+                    return player.getSecondarySubtitleStreamIndex();
+                }
+            } catch (e) {
+                console.error('[playbackmanager] Failed to get secondary stream index:', e);
+            }
+
+            if (!player) {
+                throw new Error('player cannot be null');
+            }
+
+            return getPlayerData(player).secondarySubtitleStreamIndex;
```

Comment:
Yes definitely. Since I'm new to the project, I was trying to stick to existing code style and keep my changes as minimal as I could. Going forward, I will try to go ahead and clean things up as much as I can.